nncf/torch/quantization/quantize_functions.py

```diff
@@ -259,5 +259,7 @@ def decompress(input: torch.Tensor, scale: torch.Tensor, zero_point: torch.Tenso
     :return: The decompressed tensor
     """
     input = input.type(dtype=scale.dtype)
-    decompressed_input = (input - zero_point) * scale
+    if zero_point is not None:
```

Comment:
I would suggest implementing a separate function for asymmetric and symmetric cases to make tracing easier.